Jenkinsfile_CNP

```diff
@@ -33,19 +33,17 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 GradleBuilder builder = new GradleBuilder(this, product)
 
 withPipeline(type , product, component) {
-  before('smoketest:preview') {
-    withAksClient('nonprod') {
-      // vars needed for AKS testing
-      env.TEST_S2S_NAME = 'send_letter_tests'
-      env.TEST_S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
-      env.TEST_FTP_HOSTNAME = 'cmseft.services.xerox.com'
-      env.TEST_FTP_PORT = 22
-      env.TEST_FTP_FINGERPRINT = 'SHA256:3tX3DIkqd1Loz2alHfnt+qjHocfxk0YUOZHlnf9Zgdk'
-      env.FTP_TARGET_FOLDER = 'TO_XEROX'
-      env.TEST_FTP_REPORTS_FOLDER = 'FROM_XEROX'
-      env.TEST_ENCRYPTION_ENABLED = false
-    }
-  }
+
+  // vars needed for AKS testing
+  env.TEST_S2S_NAME = 'send_letter_tests'
+  env.TEST_S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+  env.TEST_FTP_HOSTNAME = 'cmseft.services.xerox.com'
+  env.TEST_FTP_PORT = 22
+  env.TEST_FTP_FINGERPRINT = 'SHA256:3tX3DIkqd1Loz2alHfnt+qjHocfxk0YUOZHlnf9Zgdk'
+  env.TEST_FTP_REPORTS_FOLDER = 'FROM_XEROX'
+  env.TEST_ENCRYPTION_ENABLED = false
+  env.TEST_FTP_TARGET_FOLDER = 'TO_XEROX'
```

Comment:
what happened here? this one didn't exist before ( `FTP_TARGET_FOLDER` did )

Reply:
add TEST prefix to clarify and it interferes with integration test case when removing before('smoketest:preview')